govc/vm/change.go

```diff
@@ -168,6 +198,10 @@ func (cmd *change) Run(ctx context.Context, f *flag.FlagSet) error {
 		cmd.VirtualMachineConfigSpec.ExtraConfig = cmd.extraConfig
 	}
 
+	if len(cmd.extraConfigFile) > 0 {
+		cmd.VirtualMachineConfigSpec.ExtraConfig = cmd.extraConfigFile
```

Comment:
Looks like we can only use `-e` or `-f` with the same command invocation, as any `-f` would overwrite `-e` args here. How about the change below so both flags can be used?  I don't think original `len(cmd.extraConfig)` was needed.

```diff
modified   govc/vm/change.go
@@ -194,13 +194,7 @@ func (cmd *change) Run(ctx context.Context, f *flag.FlagSet) error {
 		return flag.ErrHelp
 	}
 
-	if len(cmd.extraConfig) > 0 {
-		cmd.VirtualMachineConfigSpec.ExtraConfig = cmd.extraConfig
-	}
-
-	if len(cmd.extraConfigFile) > 0 {
-		cmd.VirtualMachineConfigSpec.ExtraConfig = cmd.extraConfigFile
-	}
+	cmd.VirtualMachineConfigSpec.ExtraConfig = append(cmd.extraConfig, cmd.extraConfigFile...)
 
 	setAllocation(&cmd.CpuAllocation)
 	setAllocation(&cmd.MemoryAllocation)
```

Reply:
Applied your change :)